src/features/PatientInfo/index.js

```diff
@@ -16,39 +16,10 @@ function PatientInfo(props) {
       <h1 className="patient-name">{input.name}</h1>
       <div className="patient-detail">
         <div className="id">
-          {/* <div className="float-end"><a href="fake_ehr.html" className="view">View patient in EHR</a></div> */}
+          <div><b>Date of Birth:</b> <span><time dateTime={dob}>{dob}</time></span> ({input.age} y.o.)</div>
```

Comment:
While the patient is loading and the spinner is overlayed over the screen, none of the demographic info values are displayed yet except for "( y.o.)" in the Date of Birth field. Can we make it so that this field value is empty until the patient loads?

Reply:
I addressed this in https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/54/commits/faa1cc6e2af6b3d753c8a9f5f8b51aa0fae6a245.

Reply:
Fine with this fix.